salt/modules/vsphere.py

```diff
@@ -3224,6 +3224,19 @@ def vsan_enable(host, username, password, protocol=None, port=None, host_names=N
     return ret
 
 
+def get_dvs_portgroup_assignment(service_instance, host, dvs_name,
+                                 portgroup_name, host_names):
+    '''
+
```

Comment:
Can you add a description of what this function does, as well as a CLI function?